eng/Build.props

```diff
@@ -52,12 +52,9 @@
                       $(RepoRoot)src\Components\WebView\Samples\BlazorWpfApp\**\*.csproj;
                       " />
 
-    <!-- Projects won't build consistently. An SDK newer than 6.0.100-preview.5.21230.2 should fix the first.
-         See https://github.com/dotnet/aspnetcore/pull/32428 and https://github.com/dotnet/aspnetcore/issues/32788. -->
+    <!-- Skipping due to build unreliability with current SDK et cetera. -->
     <ProjectToExclude Include="
-                      $(RepoRoot)src\Components\benchmarkapps\Wasm.Performance\TestApp\Wasm.Performance.TestApp.csproj" />
-    <ProjectToExclude Include="
-                      $(RepoRoot)src/Components/test/E2ETestMigration/Microsoft.AspNetCore.Components.Migration.E2ETests.csproj" />    
+                      $(RepoRoot)src/Components/test/E2ETestMigration/Microsoft.AspNetCore.Components.Migration.E2ETests.csproj" />
```

Comment:
Just noticed this project is mentioned in a couple of *.slnf files but **not** in AspNetCore.sln. Should I fix that solution filter bug in this PR❔

I _think_ we disabled this project because the additional reference to src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj caused problems. Is that correct❔

Reply:
Yeah you can leave this disabled for now I will take care of adding this back once my PR is stable, currently the project fails around 33% of the time during build, so its not hard to see if it improves significantly after the SDK is updated

Reply:
In regards to the filter, if it doesn't hurt to add it to the slnf now, that'd be great

Reply:
To be clear: It's AspNetCore.sln that doesn't currently mention Microsoft.AspNetCore.Components.Migration.E2ETests.csproj and the Components.slnf and ComponentsNoDeps.slnf files _should_ be borked. Is anyone seeing issues w/ those solution filters❔

Reply:
Okay yeah I think I saw issues where the project didn't show up before by default, its because it was missing in the AspNetCore sln I guess